Lib/test/_test_multiprocessing.py

```diff
@@ -258,6 +258,9 @@ def __call__(self, *args, **kwds):
 class BaseTestCase(object):
 
     ALLOWED_TYPES = ('processes', 'manager', 'threads')
+    # If not empty, limit which start method suits run this class.
+    START_METHODS: set[str] = set()
+    start_method = None  # set by install_tests_in_module_dict()
```

Comment:
Is it necessary to initialize it here?
```suggestion
```

Reply:
I do this as a design pattern I don't like attributes on types that do not exist until some other method is run after construction.  It tells the reader what they can expect.  Since we have no `__init__` and do not want one, a class attribute makes sense.